tests/loader_wsi_tests.cpp

```diff
@@ -31,12 +31,13 @@
 #include <X11/Xutil.h>
 #endif
```

Comment:
I see that the WSI tests no longer create the actual WSI objects from the windowing system, which is smart. We aren't actually needing to use the "real deal" for testing. 
But I think this include is left over, unless its needed for something in the test I wasn't able to spot.